### ISSUE TITLE
Multiple instances of OBS WebRTC (64bit)

### ISSUE THREAD

**Issue by momi133** [CoSMoSoftware/OBS-studio-webrtc#333]
Hi Dear

is it possible to use Multiple instances of OBS WebRTC (64bit) like OBS Studio Portable?
https://obsproject.com/forum/resources/obs-and-obs-studio-portable-mode-on-windows.359/

How to use Multiple instances of OBS WebRTC (64bit) with  **Different config files**?

thanks a lot

**Comment by patrickcusack** [comment]
Yes. I've done as many as 4 at one time. As I remember when you use portable it looks for a config relative to your current path.
